rke/templates/templates.go

```diff
@@ -232,7 +233,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.22.5-rancher2-2 <1.22.9-rancher1-1":   nginxIngressv110Rancher3,
 			">=1.22.9-rancher1-1 <1.23.0-rancher1-1":   nginxIngressv120Rancher2,
 			">=1.23.0-rancher1-1 <1.23.6-rancher1-1":   nginxIngressv110Rancher3,
-			">=1.23.6-rancher1-1":                      nginxIngressv120Rancher2,
+			">=1.23.6-rancher1-1 <1.23.16-rancher1-1":  nginxIngressv120Rancher2,
+			">=1.23.16-rancher1-1":                     nginxIngressv151Rancher1,
```

Comment:
These ranges need to be updated. We still want `nginxIngressv120Rancher2` to be used for 1.24.0-1.24.9 since the new one should apply only starting from 1.24.10.